index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class="main-section">
+        <header class="header">
+            <div id="logo-text">My Logo</div>
```

Comment:
- Hey, your logo should be wrapped in an `a` tag to make it clickable as is meant to be a link that would take users to the home page which is the `index.html`. please, let's update this to improve our layout.